File: logica-general/ejercicio-029/resoluciones/lucas-pajarito/peliculasmiedo.py
peliculas = [
    {
        "titulo": "El Conjuro",
        "estado": "Pendiente"
    },
    {
        "titulo": "It",
        "estado": "Reproduciendo"
    },
    {
        "titulo": "La Monja",
        "estado": "Pausada"
    },
    {
        "titulo": "Insidious",
        "estado": "Finalizada"
    },
    {
        "titulo": "Scream",
        "estado": "Pendiente"
    }
]


def mostrar_estado_peliculas(peliculas):

    print("=== ESTADO DE LAS PELÍCULAS ===")

    for pelicula in peliculas:

        if pelicula["estado"] == "Pendiente":
            print(f'{pelicula["titulo"]}: La película aún no ha comenzado.')

        elif pelicula["estado"] == "Reproduciendo":
            print(f'{pelicula["titulo"]}: La película se está reproduciendo.')

        elif pelicula["estado"] == "Pausada":
            print(f'{pelicula["titulo"]}: La reproducción está en pausa.')

        else:
            print(f'{pelicula["titulo"]}: La película ya finalizó.')


mostrar_estado_peliculas(peliculas)

File: logica-general/ejercicio-009/resoluciones/lucas-pajarito/peliculasmiedo.js
const peliculas = [
    {
        titulo: "El Conjuro",
        estado: "Pendiente"
    },
    {
        titulo: "It",
        estado: "Reproduciendo"
    },
    {
        titulo: "La Monja",
        estado: "Pausada"
    },
    {
        titulo: "Insidious",
        estado: "Finalizada"
    },
    {
        titulo: "Scream",
        estado: "Pendiente"
    }
];

function mostrarEstadoPeliculas(peliculas) {

    peliculas.forEach(pelicula => {

        if (pelicula.estado === "Pendiente") {
            console.log(`${pelicula.titulo}: La película aún no ha comenzado.`);
        } else if (pelicula.estado === "Reproduciendo") {
            console.log(`${pelicula.titulo}: La película se está reproduciendo.`);
        } else if (pelicula.estado === "Pausada") {
            console.log(`${pelicula.titulo}: La reproducción está en pausa.`);
        } else {
            console.log(`${pelicula.titulo}: La película ya finalizó.`);
        }

    });

}

mostrarEstadoPeliculas(peliculas);